### ISSUE TITLE
Convolution operator not marked as quantizable when `padding` is defined in the class instantiation

### ISSUE THREAD

**Issue by etrommer** [google-ai-edge/ai-edge-torch#398]
### Description of the bug:

A `torch.nn.Conv2d` class instance is not marked as quantizable when padding is defined in the instantiation.

Minimal working example:
```python
import torch
import ai_edge_torch
from torch.export import export_for_training
from ai_edge_torch.quantize.pt2e_quantizer import (
    PT2EQuantizer,
    get_symmetric_quantization_config,
)
from torch.ao.quantization.quantize_pt2e import convert_pt2e, prepare_pt2e
from ai_edge_torch.quantize.quant_config import QuantConfig

def aiet_export(model: torch.nn.Module, fname: str):
    sample_input = torch.rand((1, 8, 8, 3))
    model = ai_edge_torch.to_channel_last_io(model, args=[0], outputs=[0]).eval()
    model = export_for_training(model, (sample_input,)).module()
    aiet_quantizer = PT2EQuantizer().set_global(
        get_symmetric_quantization_config(is_per_channel=False, is_dynamic=False)
    )
    model = prepare_pt2e(model, aiet_quantizer)
    with torch.no_grad():
        model(sample_input)
    model = convert_pt2e(model, fold_quantize=False)
    torch.ao.quantization.move_exported_model_to_eval(model)
    aiet_tc = ai_edge_torch.convert(
        model, (sample_input,), quant_config=QuantConfig(pt2e_quantizer=aiet_quantizer)
    )
    aiet_tc.export(fname)

class TinyConv1(EvalModule):
    def __init__(self):
        super().__init__()
        # without explicit padding definition
        self.conv = torch.nn.Conv2d(3, 2, (3, 3))

    def forward(self, x: torch.Tensor) -> torch.Tensor:
        y = self.conv(x)
        return y

class TinyConv2(EvalModule):
    def __init__(self):
        super().__init__()
        # with explicit padding definition
        self.conv = torch.nn.Conv2d(3, 2, (3, 3), padding='same')

    def forward(self, x: torch.Tensor) -> torch.Tensor:
        y = self.conv(x)
        return y

tc1 = TinyConv1()
aiet_export(tc1, 'tinyconv1.tflite')

tc2 = TinyConv2()
aiet_export(tc2, 'tinyconv2.tflite')

```

## Expected:
Equally quantized models

## Actual:
TinyConv1 has quantized parameters:
![gh_aiet_tc1](https://github.com/user-attachments/assets/915e910a-362e-4aad-8554-e4c4d014327b)

TinyConv2 does not:
![gh_aiet_tc2](https://github.com/user-attachments/assets/69b5bcf2-a842-45b4-89c6-995dbc37c54e)

### Any other information you'd like to share?

The cause of this issue is that the test for identifying a `conv` node in the FX graph when applying quantization is too restrictive:
https://github.com/google-ai-edge/ai-edge-torch/blob/d4e358e15174c23b33026bd13659523f91d7a1e0/ai_edge_torch/quantize/pt2e_quantizer_utils.py#L311-L315

When padding is defined in the instantiation, the FX graph node becomes an instance of `torch._ops.OpOverload`, so this test fails even though it probably shouldn't.

**Comment by etrommer** [comment]
If you accept PRs, I'm happy to open one for this.

**Comment by pkgoogle** [comment]
Hi @etrommer, this repo is open for contribution, please do so and we will review 😃

**Comment by aravindmurali22** [comment]
Hi @etrommer ,

Thank you for taking the time to report this and for your patience. Please let us know if you are experiencing this issue in the latest [release](https://github.com/google-ai-edge/litert-torch/releases/tag/v0.9.1) of litert-torch. If the problem persists, please file a new issue with the updated details. This will help us track and investigate it using the most current context.

We will close this issue to keep our tracker aligned with the current codebase and release cycle. 
Thank you again for your support and understanding!
